packages/fluentui-rc/src/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I don't think this change is needed?

Reply:
it is not needed for the fixing types for `labelValue`

just noticed I missed it when rebasing typescript project references branch(this is needed to have correct ts project references for test folder)